hack/package-helm-charts.sh

```diff
@@ -16,6 +16,13 @@
 
 set -o pipefail
 
+# if arg1 is set, it will be used as the version number
+if [ -z "$1" ]; then
+  VERSION=$(git describe --tags --always --dirty)
+else
+  VERSION=$1
+fi
```

Comment:
Is it simpler to make this a required argument? Alternatively should we extract the version from `versions.mk`?